.github/workflows/ci.yml

```diff
@@ -0,0 +1,24 @@
+name: Continuous Integration
+on: push
+jobs:
+  lint:
+    runs-on: ubuntu-latest
```

Comment:
As long as you only run docker images I guess the risk is low, but if you want to do anything on the host OS directly, I'd suggest pinning it. I noticed when looking at Dependabot that there's a mode for GitHub Actions, so I expecting bumping pinned deps will soon be automated, if it isn't already working.